README.md

```diff
@@ -137,6 +137,7 @@
 - [BetterZip](http://macitbetter.com/) - A very capable and full-featured archive manager.
 - [DaisyDisk](http://www.daisydiskapp.com/) - Analyze disk usage and free up disk space.
 - [Finicky](https://johnste.github.io/finicky/) - App that allows you to set rules that decide which browser is opened for every link. [![Open-Source Software][OSS Icon]](https://github.com/johnste/finicky)
+- [GrandPerspective](http://grandperspectiv.sourceforge.net/) - Graphically show the disk usage within a file system. [![Open-Source Software][OSS Icon]](http://sourceforge.net/p/grandperspectiv/code/HEAD/tree/)
```

Comment:
I feel DaisyDisk is much better than this as it not only monitors disk usage but also cleans up the disk and has a variety of features, also looks like GrandPerspective is pretty slow on development too, it was last updated more than a year ago.